frontend/public/components/secscan/donut-chart/donut-chart.jsx

```diff
@@ -1,100 +0,0 @@
-import * as _ from 'lodash-es';
-import * as React from 'react';
-import * as ReactDOM from 'react-dom';
-
-import * as d3 from 'd3';
-
-// Donut chart
```

Comment:
I think we need to keep this for topology view

Reply:
Actually... It looks like we have two donut charts?

https://github.com/openshift/console/blob/master/frontend/public/components/graphs/donut.jsx